src/java/org/apache/cassandra/metrics/RepairMetrics.java

```diff
@@ -18,17 +18,41 @@
 
 package org.apache.cassandra.metrics;
 
+import java.util.Collections;
+import java.util.EnumMap;
+import java.util.Map;
+
 import com.codahale.metrics.Counter;
+import com.codahale.metrics.Histogram;
+import org.apache.cassandra.net.Verb;
 
 import static org.apache.cassandra.metrics.CassandraMetricsRegistry.Metrics;
 
 public class RepairMetrics
 {
     public static final String TYPE_NAME = "Repair";
     public static final Counter previewFailures = Metrics.counter(DefaultNameFactory.createMetricName(TYPE_NAME, "PreviewFailures", null));
+    private static final Histogram retries = Metrics.histogram(DefaultNameFactory.createMetricName(TYPE_NAME, "Retries", null), false);
```

Comment:
Do we need to write tests for these metrics?

Reply:
wrote tests and found a bug!  thanks for pushing me to do it!

The bug was that a failed request would trigger updating the timeout metric, so an `UNKNOWN` would increment `TIMEOUT` metric...  trivial fix, but tests helped to point that out